xml/deployment_cifs.xml

```diff
@@ -215,14 +215,34 @@
     <link xlink:href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-install-quick/"/>.
    </para>
    <procedure xml:id="proc-sec-ses-cifs-ha">
+
     <step>
      <para>
       Create a &sgw; specific keyring on the &adm; and copy it to both nodes:
      </para>
 <screen>&prompt.cephuser;<command>ceph</command> auth get-or-create client.samba.gw mon 'allow r' \
     osd 'allow *' mds 'allow *' -o ceph.client.samba.gw.keyring
-&prompt.smaster;<command>scp</command> ceph.client.samba.gw.keyring &si.earth;:/etc/ceph/
-&prompt.smaster;<command>scp</command> ceph.client.samba.gw.keyring &si.mars;:/etc/ceph/</screen>
+&prompt.cephuser;<command>scp</command> ceph.client.samba.gw.keyring &si.earth;:/etc/ceph/
+&prompt.cephuser;<command>scp</command> ceph.client.samba.gw.keyring &si.mars;:/etc/ceph/</screen>
+    </step>
+    <step>
+     <para>
+      SLE-HA setup requires a fencing device for split brain avoidance.  For
```

Comment:
```suggestion
      SLE-HA setup requires a fencing device for split brain avoidance. For

Reply:
I have no idea what "split brain avoidance" means?!

Reply:
I think it makes sense to refer to the SLE-HA documentation, where possible. For "split brain" they have a link to an external site:
https://techthoughts.typepad.com/managing_computers/2007/10/split-brain-quo.html